bigtable-hbase/pom.xml

```diff
@@ -44,28 +44,6 @@ limitations under the License.
         </dependency>
 
         <dependency>
-            <groupId>com.google.api-client</groupId>
```

Comment:
There were 3 duplicated dependencies that mvn deploy complained about.  I had to remove them for mvn deploy to work.